addons/mail/static/src/components/message/message.js

```diff
@@ -104,6 +104,9 @@ class Message extends Component {
     _constructor() {}
 
     mounted() {
+        for(const el of [...this._contentRef.el.querySelectorAll(':scope .o_Message_readMore')]) {
+            el.remove();
+        }
```

Comment:
Whitespace between `for` and `(`.
Also put a short comment above on why this is needed.